proc_nutation.py

```diff
@@ -1,53 +1,84 @@
+from pylab import *
 from pyspecdata import *
 from scipy.optimize import minimize
-from proc_scripts import hermitian_function_test,zeroth_order_ph,postproc_dict,fl_mod,center_CPMG_echo
+from proc_scripts import hermitian_function_test,zeroth_order_ph,postproc_dict,fl_mod
 from sympy import symbols
 from numpy import *
 fl = fl_mod()
 t2 = symbols('t2')
 logger = init_logging("info")
-for searchstr,exp_type,nodename,postproc in [
-    ['200219_nutation_alex_probe','test_equip','nutation','spincore_nutation_v1']
+max_kHz = 200
+for searchstr,exp_type,nodename,postproc,freq_slice,p90_varied in [
```

Comment:
you should be able to detect whether or not p90 is varied based on the naming of the dimensions, so you shouldn't need to specify it here.

Reply:
nice!